frontend/packages/pipelines-plugin/src/utils/pipeline-actions.tsx

```diff
@@ -208,6 +210,43 @@ export const stopPipelineRun: KebabAction = (kind: K8sKind, pipelineRun: Pipelin
   };
 };
 
+export const cancelPipelineRunFinally: KebabAction = (
```

Comment:
pipeline-actions-spec.tsx has test for stop action, could you please add on for this `cancel` action as well.